internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -826,7 +826,10 @@ components:
       type: object
       required:
         - region
-        - share_with_accounts
```

Comment:
I wonder if it would not be better to have `oneOf` block here and specify the two variants of the required fields. It is in both cases the `region`, but in one it would be `share_with_accounts` and on the second the `public` field.